src/python/include/Alpha_complex_interface.h

```diff
@@ -60,7 +60,7 @@ class Alpha_complex_interface {
   }
 
   void create_simplex_tree(Simplex_tree_interface<>* simplex_tree, double max_alpha_square) {
-    alpha_complex_->create_complex(*simplex_tree, max_alpha_square);
+    alpha_complex_->create_complex(*simplex_tree, max_alpha_square, false, false);
```

Comment:
It doesn't hurt, but it wasn't necessary, and could cause conflicts if you are doing the Epeck_d thing at the same time.

Reply:
Yes you are right. I rolled back this modification on 7cd38c50d3843fd222d4210507bad948afe63912